lib/python/src/bailo/helper/model.py

```diff
@@ -182,6 +194,10 @@ def get_latest_release(self):
         releases = self.get_releases()
         if releases == []:
             raise BailoException("This model has no releases.")
+
+        latest_release = max(releases)
+        logger.info(f"Latest release ({str(latest_release.version)}) for {self.model_id} retrieved successfully.")
+
         return max(releases)
```

Comment:
Could we change this to `latest_release`?

Reply:
Done